### PR TITLE
Change all [user] to user[X]

### PR DESCRIPTION
**Pull request by chrira**
unify user placeholder

### CODE REVIEW

File: labs/322_install_openshift.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ member 92c764a37c90869 is healthy: got healthy result from https://172.31.46.235
 cluster is healthy
 
 [ec2-user@master0 ~]# sudo etcdctl -C https://master0.user[X].lab.openshift.ch:2379 --ca-file=/etc/origin/master/master.etcd-ca.crt --cert-file=/etc/origin/master/master.etcd-client.crt --key-file=/etc/origin/master/master.etcd-client.key member list
-92c764a37c90869: name=master0.user[X].lab.openshift.ch peerURLs=https://172.31.46.235:2380 clientURLs=https://172.31.46.235:2379 isLeader=true
```
Comment (bliemli) [inline labs/322_install_openshift.md]: This is correct.

File: labs/331_user_management.md [hunk 2]
```diff
@@ -44,8 +44,8 @@ cluster role "cluster-admin" added: "test-user"
 ```
 Now you can try to login from your client with the user and check if you see all projects
 ```
-[localuser@localhost ~]$ oc login https://console.[user].lab.openshift.ch
-Authentication required for https://console.user6.lab.openshift.ch (openshift)
+[localuser@localhost ~]$ oc login https://console.user[X].lab.openshift.ch
+Authentication required for https://console.userX.lab.openshift.ch (openshift)
```
Comment (bliemli) [inline labs/331_user_management.md]: Not consistent with the other changes.

File: labs/332_update_host.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ First, let's check if the excludes are set by Openshift on all nodes. Connect to
 ```
 [ec2-user@master0 ~]$ ansible all -m shell -a "grep exclude /etc/yum.conf"
 ...
-node1.user[X].lab.openshift.ch | SUCCESS | rc=0 >>
```
Comment (bliemli) [inline labs/332_update_host.md]: This is correct.

File: labs/332_update_host.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ First login as cluster-admin and drain the first node.
 If you list all running pods on the node you should see, that just the DaemonSets (glusterfs-storage and logging-fluentd) are left.
 ```
 [ec2-user@master0 ~]$ oc adm manage-node node1.user[X].lab.openshift.ch --list-pods
-Listing matched pods on node: node1.user6.lab.openshift.ch
+Listing matched pods on node: node1.userX.lab.openshift.ch
```
Comment (bliemli) [inline labs/332_update_host.md]: Not consistent with the other changes.

File: labs/332_update_host.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ Let's check if the node is unscheduled.
 ```
 [ec2-user@master0 ~]$ oc get nodes
 ...
-node1.user[X].lab.openshift.ch     Ready,SchedulingDisabled   2d        v1.6.1+5115d708d7
```
Comment (bliemli) [inline labs/332_update_host.md]: This is correct.

File: labs/333_persistent_storage.md [hunk 6]
```diff
@@ -74,7 +74,7 @@ If you create a pvc, Heketi will automatically create a pv and bind it to your p
 Create a new project, set default storageclass and create a pvc
 ```
 [ec2-user@master0 ~]$ oc new-project test
-Now using project "test" on server "https://console.[user].lab.openshift.ch:8443".
+Now using project "test" on server "https://console.userX.lab.openshift.ch:8443".
```
Comment (bliemli) [inline labs/333_persistent_storage.md]: Not consistent with the other changes.

File: labs/351_backup.md [hunk 7]
```diff
@@ -139,23 +139,23 @@ The cluster is now initialized, so we need to remove the "--force-new-cluster" p
...
+[root@master0 ~]# etcdctl -C https://master0.user[X].lab.openshift.ch:2379 --ca-file=/etc/etcd/ca.crt --cert-file=/etc/etcd/peer.crt --key-file=/etc/etcd/peer.key ls /
 /openshift.io
 ```
 
 we need to change the peerURL of the etcd to it's private ip.
 ```
-[root@master0 ~]# etcdctl -C https://master0.[user].lab.openshift.ch:2379 --ca-file=/etc/etcd/ca.crt --cert-file=/etc/etcd/peer.crt --key-file=/etc/etcd/peer.key member list
-92c764a37c90869: name=master0.[user].lab.openshift.ch peerURLs=https://localhost:2380 clientURLs=https://[private_ip]:2379 isLeader=true
+[root@master0 ~]# etcdctl -C https://master0.user[X].lab.openshift.ch:2379 --ca-file=/etc/etcd/ca.crt --cert-file=/etc/etcd/peer.crt --key-file=/etc/etcd/peer.key member list
+92c764a37c90869: name=master0.userX.lab.openshift.ch peerURLs=https://localhost:2380 clientURLs=https://[private_ip]:2379 isLeader=true
```
Comment (bliemli) [inline labs/351_backup.md]: Not consistent with the other changes.

File: labs/351_backup.md [hunk 8]
```diff
@@ -139,23 +139,23 @@ The cluster is now initialized, so we need to remove the "--force-new-cluster" p
...
 
-[root@master0 ~]# etcdctl -C https://master0.[user].lab.openshift.ch:2379 --ca-file=/etc/etcd/ca.crt --cert-file=/etc/etcd/peer.crt --key-file=/etc/etcd/peer.key member update 6248d01c5701 https://[private_ip]:2379
+[root@master0 ~]# etcdctl -C https://master0.user[X].lab.openshift.ch:2379 --ca-file=/etc/etcd/ca.crt --cert-file=/etc/etcd/peer.crt --key-file=/etc/etcd/peer.key member update 6248d01c5701 https://[private_ip]:2379
 Updated member with ID 6248d01c5701 in cluster
 
-[root@master0 ~]# etcdctl -C https://master0.[user].lab.openshift.ch:2379 --ca-file=/etc/etcd/ca.crt --cert-file=/etc/etcd/peer.crt --key-file=/etc/etcd/peer.key member list
-6248d01c5701: name=master0.[user].lab.openshift.ch peerURLs=https://172.31.46.201:2379 clientURLs=https://172.31.46.201:2379 isLeader=true
+[root@master0 ~]# etcdctl -C https://master0.user[X].lab.openshift.ch:2379 --ca-file=/etc/etcd/ca.crt --cert-file=/etc/etcd/peer.crt --key-file=/etc/etcd/peer.key member list
+6248d01c5701: name=master0.userX.lab.openshift.ch peerURLs=https://172.31.46.201:2379 clientURLs=https://172.31.46.201:2379 isLeader=true
```
Comment (bliemli) [inline labs/351_backup.md]: Not consistent with the other changes.

**Review by bliemli — CHANGES_REQUESTED** [review]

**Comment by chrira** [comment]
changed all userX to user[X] and rebased again

**Pull request merged**